bin/rucio

```diff
@@ -918,12 +918,26 @@ def upload(args):
     trace['hostname'] = socket.getfqdn()
     trace['scope'] = fscope
     trace['uuid'] = generate_uuid()
+    if len(files) > 1 and args.guid:
+        logger.error("A single GUID was specified on the command line, but there are multiple files to upload.")
+        logger.error("If GUID auto-detection is not used, only one file may be uploaded at a time")
+        return FAILURE
+    if len(files) > 1 and args.lfn:
```

Comment:
To be consistent with rucio terminology lfn should be renamed to `name`. And the parts below which use the variable `name` should be renamed to `filename`. Thus if no --name is provided, the base_name of the filename is used as name, otherwise the explicitly defined name. This should also be consistent throughout the protocols, as now the name is not necessarily the filename anymore.